test/Hydration-test.js

```diff
@@ -0,0 +1,110 @@
+import dayjs from 'dayjs'
+import { expect } from 'chai'
+import Hydration from '../src/Hydration'
+
+describe('Hydration', () => {
+  let userHydrationData1
+  let userHydrationData2
+  let userHydrationData3
+  let userHydrationData4
+  let userHydrationData5
+  let userHydrationData6
+  let userHydrationData7
+  let userHydrationData8
+  let testFilteredData
+  let hydratedUser
+
+  beforeEach(() => {
+    userHydrationData1 = {
+      date: 'Jun 19 2019',
+      numOunces: 85,
+    }
+
+    userHydrationData2 = {
+      date: 'Jun 20 2019',
+      numOunces: 69,
+    }
+
+    userHydrationData3 = {
+      date: 'Jun 21 2019',
+      numOunces: 54,
+    }
+
+    userHydrationData4 = {
+      date: 'Jun 22 2019',
+      numOunces: 41,
+    }
+
+    userHydrationData5 = {
+      date: 'Jun 23 2019',
+      numOunces: 37,
+    }
+
+    userHydrationData6 = {
+      date: 'Jun 24 2019',
+      numOunces: 75,
+    }
+
+    userHydrationData7 = {
+        date: 'Jun 25 2019',
+        numOunces: 47
+    }
+
+    userHydrationData8 = {
+      date: 'Jun 26 2019',
+      numOunces: 99,
+    }
```

Comment:
If interested, I can help with putting your test data in another file to clean this up, but if that's too much to take on right now, that's also ok :)